src/com/github/alex_moon/porcupi/Teller.java

```diff
@@ -0,0 +1,8 @@
+package com.github.alex_moon.porcupi;
+
+import com.github.alex_moon.porcupi.messages.Message;
+
+public interface Teller {
```

Comment:
`Tellable` - `Teller` sounds like a bank employee, and more importantly this interface says nothing about `tell`ing any other module, only that it can be `tell`ed.